pegasus-spark/.travis.yml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I think this file is useless because we dodn't use Travis for CI, instead, we use Github actions.